code/modules/clothing/suits/ego_gear/realized.dm

```diff
@@ -174,7 +174,7 @@ No Ability	250
 	name = "solemn eulogy"
 	desc = "Death is not extinguishing the light, it is putting out the lamp as dawn has come."
 	icon_state = "eulogy"
-	armor = list(RED_DAMAGE = 40, WHITE_DAMAGE = 80, BLACK_DAMAGE = 80, PALE_DAMAGE = 50)		//No ability
+	armor = list(RED_DAMAGE = 40, WHITE_DAMAGE = 70, BLACK_DAMAGE = 70, PALE_DAMAGE = 30)		//Ranged
```

Comment:
We actually are now rebalancing all realized armor to 230.

Reply:
so do I re-buff it? it's got 210s because it's ranged

Reply:
Please do. I'll re-buff it now and then merge.